### ISSUE TITLE
After completed installation, nothing chaged...

### ISSUE THREAD

**Issue by Zapelmon** [gh0stzk/dotfiles#294]
**THIS IS MY PROBLEM, NOT A DEV OR PROGRAM ISSUE**
I am a **new user** to Arch Linux and I tried to read all the information from wiki to solve my problem, but my brain didn't retain any useful details. After I ran ./RiceInstaller and followed all the necessary steps, nothing changed. Bspwm remained at the basic settings I configured in chroot after installation (for the ~40th time), so it's even starts. I think due to my **inexperience**, I'll need to reinstall the system again, but I want to know how to install it correctly. Maybe I messed up in the last section of the installer when choosing options using numbers. 

**None** of the keybinds **are** **working**, including those listed under "_Very useful keybindings to know..._" as well as the F1, F2, F3... keys. I'm using a laptop with a 1600x900 resolution display at 96dpi. The configuration should work correctly, but... there are no configs...

I'm sorry for being such a **noob**, but I can't install it because of skill issue. I've been practicing installing Arch manually all week, only to get stuck on installing a simple installer for themes and settings for bspwm. 
Probably switching my hostname from I-Use-Arch-BTW to I-Am-Dumb-BTW......

**Comment by gh0stzk** [comment]
Ok you run my script in chroot?

**Comment by gh0stzk** [comment]
If that is the case, your configs are in root, and that doesnt be good. You need to run my script in your new installation inside your new login, not root.

**Comment by Zapelmon** [comment]
> If that is the case, your configs are in root, and that doesnt be good. You need to run my script in your new installation inside your new login, not root.

I apologize for any misunderstandings; English is not my native language. No, I did not run your script in chroot. I was saying that I barely managed to get everything running, and I configured the basic bspwm setup in chroot just to make it start. The script was executed as a user (superuser, without using `sudo -i` beforehand). I tried reinstalling the system using archinstall in case I messed up the initial installation, but the result didn't change. There's a high chance I'm doing something wrong, but I don't understand what.

After installing everything and entering the sudo password everywhere, when I reach the settings menu, I essentially look at what changes are being suggested and exit by pressing 0 everywhere. In short, while I may be an idiot, I'm not so much of one as to run a user-intended script in chroot. However, maybe that's precisely why the script couldn't do anything.

List of all commands I used to configure bspwm, maybe I moved files to the wrong place and the script didn't change anything:

```bash
sudo pacman -S polybar picom dmenu nitrogen alacritty thunar
cd /home/lenovo/.config
mkdir bspwm sxhkd polybar picom
cp /usr/share/doc/pspwm/examples/bspwmrc bspwm/
cp /usr/share/doc/sxhkd/examples/sxhkdrc sxhkd/
cp /etc/xdg/picom.conf picom/
cp /etc/polybar/config.ini polybar/
cp /etc/stuntsrc stunts/
chmod +x bspwm/bspwmrc
nano bspwm/bspwmrc
cd sxhkd
nano sxhkdrc
```
(in nano im only edited autolaunching of picom polybar nitrogen, and in nano sxhkdrc changed terminal)

**Comment by AlvinPix** [comment]
Don't tell yourself that you're an idiot, you just don't have the knowledge that's needed. I recommend that you re-install arch linux from scratch. You can use "arch install" or tutorials on youtube. Our beloved ghost has automated installation scripts for installing arch, in case you don't want to use "arch install".

Lastly, it's better to configure things in your user folder than in the root folder, that can cause problems. There are many tutorials, it's just a matter of researching and you'll get it.

**Comment by gh0stzk** [comment]
Ye, make ur arch installation, then, there is no need to create a bspwm config before or having and environment, just login in your account, and follow the steps

`curl -L https://is.gd/gh0stzk_dotfiles -o $HOME/RiceInstaller`

`chmod +x RiceInstaller`

`./RiceInstaller`

The installer installs the necessary packages and dependencies. you should be fine. then install a login manager... there are to many, i use lightdm or xinitrc and login into bspwm.

**Comment by Zapelmon** [comment]
Okay, as I understand it, I can install Arch Linux without any desktop environment and just run a script? If that works, it would be great. So, I’ve additionally checked out the Arch Wiki and filesystem structure, as well as bspwm documentation, and it seems that the configs I made in chroot could have moved the files. Sorry again, I'm really new to Arch Linux. One thing I can say is that the community here is much better than in Debian, where people almost roasted me for my similar question 😄. Closing the issue and going to reinstall the system without a desktop environment, and will just run the script. Btw, using ly login manager, because it's cool and minimalistic.
